src/SerialQueue.csproj

```diff
@@ -2,16 +2,24 @@
 
   <PropertyGroup>
     <TargetFrameworks>netstandard2.0;net462</TargetFrameworks>
-    <AssemblyVersion>2.0.0.0</AssemblyVersion>
-    <FileVersion>2.0.0.0</FileVersion>
+    <AssemblyVersion>2.1.0.0</AssemblyVersion>
+    <FileVersion>2.1.0.0</FileVersion>
     <LangVersion>8.0</LangVersion>
-    <Version>2.0.0</Version>
+    <Version>2.1.0</Version>
     <Authors>Orion Edwards</Authors>
     <Company />
-    <PackageLicenseUrl>https://raw.githubusercontent.com/borland/SerialQueue/master/LICENSE</PackageLicenseUrl>
+    <PackageLicenseUrl></PackageLicenseUrl>
     <PackageProjectUrl>https://github.com/borland/SerialQueue</PackageProjectUrl>
     <Description>C# Implementation of a SerialQueue in the style of Apple's Grand Central Dispatch queues.</Description>
     <Copyright>Orion Edwards 2020</Copyright>
+    <PackageLicenseFile>LICENSE</PackageLicenseFile>
```

Comment:
Is it better to use licence expressions than files? I'm not sure.
https://docs.microsoft.com/en-us/nuget/reference/msbuild-targets#packing-a-license-expression-or-a-license-file

Reply:
I'll stick with a file for now